### PR TITLE
OSDOCS-3583: Release notes for OCP on ARM

### PR DESCRIPTION
**Pull request by kelbrown20**
For version 4.11
Jira story: [OSDOCS-3508](https://issues.redhat.com//browse/OSDOCS-3508)

Preview:[
OpenShift Container Platform 4.11 release notes -> Installation and upgrade](https://kelbrown20.github.io/OCP-on-arm-doc-previews-4.11/OSDOCS-3508-arm-release-notes-4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-installation-and-upgrade)

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -74,6 +74,25 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
...
 
+[id="ocp-4-11-OCP-on-arm"]
+==== {product-title} on ARM
+
+{product-title} 4.11 is now supported on ARM architecture based AWS user-provisioned infrastructure and bare-metal installer-provisioned infrastructure. Instance availability and installation documentation can be found in xref:../installing/installing-preparing.html#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms].
+
+The following features are supported for {product-title} on ARM:
+
+* Disconnected installation support
+* Red Hat Runtimes
```
Comment (yselkowitz) [inline release_notes/ocp-4-11-release-notes.adoc:85]: Most runtimes are not currently available for ARM, and I'm not sure of the timelines of their possible enablement.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -74,6 +74,25 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
...
+* OpenShift Pipelines
+* OpenShift Servicemesh
```
Comment (yselkowitz) [inline release_notes/ocp-4-11-release-notes.adoc:87]: Service Mesh is still unscheduled and will certainly not be available for ARM by or around the time of 4.11 GA.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -74,6 +74,25 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
...
+* Elastic file system (EFS) for AWS
+* Local storage operator on bare metal
+* Internet small computer systems (iSCSI) for bare metal
+
+The following Operators are supported for {product-title} on ARM
+
+* Special resource operator (SRO)
```
Comment (yselkowitz) [inline release_notes/ocp-4-11-release-notes.adoc:94]: Should the various operators in RHACM 2.5 be listed here?

Comment (kelbrown20) [reply release_notes/ocp-4-11-release-notes.adoc:94]: Not sure actually, but I'll check that out, thank you! Is there a list of these operators anywhere?

File: release_notes/ocp-4-11-release-notes.adoc [hunk 4]
```diff
@@ -74,6 +74,23 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
...
 
+[id="ocp-4-11-OCP-on-arm"]
+==== {product-title} on ARM
+
+{product-title} 4.11 is now supported on ARM architecture based AWS user-provisioned infrastructure and bare-metal installer-provisioned infrastructure. Instance availability and installation documentation can be found in xref:../installing/installing-preparing.html#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms].
+
+The following features are supported for {product-title} on ARM:
+
+* Disconnected installation support
+* OpenShift Pipelines
```
Comment (jeffdyoung) [inline release_notes/ocp-4-11-release-notes.adoc:85]: @kelbrown20 Pipelines won't release until Septemberish. Can you remove this line?

File: release_notes/ocp-4-11-release-notes.adoc [hunk 5]
```diff
@@ -74,6 +74,22 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
...
+==== {product-title} on ARM
+
+{product-title} 4.11 is now supported on ARM architecture based AWS user-provisioned infrastructure and bare-metal installer-provisioned infrastructure. Instance availability and installation documentation can be found in xref:../installing/installing-preparing.html#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms].
+
+The following features are supported for {product-title} on ARM:
+
+* Disconnected installation support
+* Elastic file system (EFS) for AWS
+* Local storage operator on bare metal
+* Internet small computer systems (iSCSI) for bare metal
```
Comment (aleskandro) [inline release_notes/ocp-4-11-release-notes.adoc:87]: hi @kelbrown20 should this be `Internet Small Computer Systems Interface`

**Comment by aleskandro** [comment]
Refers ARMOCP-375 ARMOCP-379

File: release_notes/ocp-4-11-release-notes.adoc (hunk 5 above)
Comment (kelbrown20) [reply release_notes/ocp-4-11-release-notes.adoc:87]: Yes, thank you!

File: release_notes/ocp-4-11-release-notes.adoc [hunk 6]
```diff
@@ -74,6 +74,22 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
...
+{product-title} 4.11 is now supported on ARM architecture based AWS user-provisioned infrastructure and bare-metal installer-provisioned infrastructure. Instance availability and installation documentation can be found in xref:../installing/installing-preparing.html#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms].
+
+The following features are supported for {product-title} on ARM:
+
+* Disconnected installation support
+* Elastic file system (EFS) for AWS
+* Local storage operator on bare metal
+* Internet Small Computer Systems Interface (iSCSI) for bare metal
+
+The following Operators are supported for {product-title} on ARM
```
Comment (stevsmit) [inline release_notes/ocp-4-11-release-notes.adoc:89]: Do we need a colon after ARM here?

Comment (kelbrown20) [reply release_notes/ocp-4-11-release-notes.adoc:89]: Ooo, yes, missed that one

**Comment by kelbrown20** [comment]
@jeffdyoung @andymcc Hello! Can you guys take one last look at the release notes?

**Pull request merged**